### PR TITLE
[CodeGen] Regen some old tests; NFC

### PR DESCRIPTION
**Pull request by goldsteinn**

### CODE REVIEW

File: llvm/test/CodeGen/WebAssembly/conv.ll [hunk 1]
```diff
@@ -1,3 +1,4 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 4
```
Comment (aheejin) [inline llvm/test/CodeGen/WebAssembly/conv.ll:1]: Assertions in this test and the below (`simd-sext-inreg.ll`) are not autogenerated. Why are these said to be autogenerated?

File: llvm/test/CodeGen/AMDGPU/store-private.ll [hunk 2]
```diff
@@ -16,6 +17,47 @@
 
 ; SI: buffer_store_byte
 define amdgpu_kernel void @store_i1(ptr addrspace(5) %out) {
+; EG-LABEL: store_i1:
```
Comment (arsenm) [inline llvm/test/CodeGen/AMDGPU/store-private.ll:20]: This one at least is pretty broken. It didn't generate the relevant checks and only the R600 lines.

We probably shouldn't have any generated tests with shared r600 and gcn run lines

File: llvm/test/CodeGen/AArch64/arm64-narrow-st-merge.ll [hunk 3]
```diff
@@ -7,6 +8,19 @@
 ; CHECK-STRICT: strh wzr
 ; CHECK-STRICT: strh wzr
```
Comment (RKSimon) [inline llvm/test/CodeGen/AArch64/arm64-narrow-st-merge.ll:9]: The script won't touch checks outside actual methods - you will need to remove these manually.

**Review by RKSimon — COMMENTED** [review]
It looks like a lot of these files need some manual cleanup

File: llvm/test/CodeGen/SystemZ/int-mul-10.ll [hunk 4]
```diff
@@ -6,10 +7,6 @@ declare i64 @foo()
 
 ; Check sign-extended multiplication in which only the high part is used.
 define i64 @f1(i64 %dummy, i64 %a, i64 %b) {
-; CHECK-LABEL: f1:
-; CHECK-NOT: {{%r[234]}}
-; CHECK: mgrk %r2, %r3, %r4
-; CHECK: br %r14
```
Comment (uweigand) [inline llvm/test/CodeGen/SystemZ/int-mul-10.ll:12]: This looks broken, it just removes all the actual checks in this test case?

**Comment by goldsteinn** [comment]
Closing this, no longer seem relevent to the original commit.

**Pull request closed without merging**